pfetch

```diff
@@ -113,6 +113,9 @@ get_os() {
             # don't follow any os-release/lsb standards whatsoever.
             command -v crux && distro=$(crux)
             command -v guix && distro='Guix System'
+            [ -e /bedrock/etc/bedrock-release ] &&
+                echo $PATH | grep -q /bedrock/cross/ &&
+                distro='Bedrock Linux'
```

Comment:
This entire block can be replaced with:

```sh
case $PATH in
    */bedrock/cross/*) distro='Bedrock Linux'
esac
```

We may be able to skip checking for the `bedrock-release` file as this path entry shouldn't exist on non-bedrock systems. If you'd still like to add it, that's fine by me.

Reply:
I _knew_ you'd prefer a pure shell implementation and experimented with some alternatives before falling back to calling the external `grep`, but using `case` skipped my mind.  A-okay with me.

I can imagine false-positive scenarios from skipping the file check, but none are sufficiently likely to clearly justify the complexity and code cost given emphasis on simplicity here.  I find this perfectly acceptable as well.